internal/config/config.go

```diff
@@ -141,23 +141,21 @@ func Read(configFile string) (*Config, error) {
 		return nil, fmt.Errorf("could not read config file: %w", err)
 	}
 
+	c := &Config{}
+	if err := yaml.Unmarshal(b, c); err != nil {
+		return nil, fmt.Errorf("could not parse config file: %w", err)
+	}
+
 	restCfg, err := ctrl.GetConfig()
 	if err != nil {
 		return nil, fmt.Errorf("could not get kubernetes REST config: %v", err)
 	}
 
-	c := &Config{
-		Scheme:     runtime.NewScheme(),
-		RestConfig: restCfg,
-		Egress: &Egress{
-			HTTPClientTimeout:       metav1.Duration{Duration: HTTPClientDefaultTimeout},
-			SnykAPIBaseURL:          SnykAPIDefaultBaseURL,
-			SnykServiceAccountToken: os.Getenv("SNYK_SERVICE_ACCOUNT_TOKEN"),
-		},
-	}
-
-	if err := yaml.Unmarshal(b, c); err != nil {
-		return nil, fmt.Errorf("could not unmarshal config file: %w", err)
+	c.RestConfig = restCfg
+	c.Egress = &Egress{
+		HTTPClientTimeout:       metav1.Duration{Duration: HTTPClientDefaultTimeout},
+		SnykAPIBaseURL:          SnykAPIDefaultBaseURL,
+		SnykServiceAccountToken: os.Getenv("SNYK_SERVICE_ACCOUNT_TOKEN"),
```

Comment:
I'm not quite sure why you changed this, as far as I can tell now user's Egress setting will be overwritten by the `c.Egress = &Egress{...}` part.

I think it should be exactly as it was before? 🤔 Create a "default" values & then unmarshal into it.

Reply:
> Create a "default" values & then unmarshal into it.

Yeah, rolling that now. 👍